pkgs/development/tools/icestorm/default.nix

```diff
@@ -1,19 +1,19 @@
-{ stdenv, fetchFromGitHub, python3, libftdi }:
+{ libcxxStdenv, fetchFromGitHub, python3, libftdi }:
 
-stdenv.mkDerivation rec {
+libcxxStdenv.mkDerivation rec {
```

Comment:
since commit https://github.com/cliffordwolf/icestorm/commit/b9e6d07e9ca9c4c6fb758e6886f43684948d70a7 clang isn't optional anymore. you can suggest a better solution if you like ;)